### ISSUE TITLE
accuracy

### ISSUE THREAD

**Issue by huuub** [mKeRix/room-assistant#289]
I have a raspberry pi zero behind the bed (from wood).
The only thing I want to know is how many persons are in the bed. 
This is my config

bluetoothClassic:
  addresses:
    - '50:77:05:B7:...'
    - '14:9F:3C:41:....'
  minRssi: 0
  interval: 10

Still it is impossible to get an accurate reading for it. the room under it is the living room. If I sit there it also gives me a false positive sometimes. On the other hand when I am in the bed (and 1 meter from the RPZ) it sometimes says I am not in the room. 
In other words, this is not accurate enough/working?

**Comment by mKeRix** [comment]
How long do the wrong readings last usually - are those just small blimps or longer phases?

Generally accuracy is tricky, since it depends on so many different factors. For multi-level homes there is an open issue with the idea of improving accuracy by also evaluation other nodes on the same level. Possibly that could also help in your case, if it just jumps between the two rooms that are on top of each other. Alternatively you could make use of the debouncing settings or maybe wait for the rolling average feature request (#285).

**Comment by huuub** [comment]
Well I already thought about waiting at least 5 minutes. Which will work a lot better (but still not always perfect). But I let my alarm/lights out depend on it. So I just want it to be correct.
